dssat_files/dssat_input/DSSAT48.INH

```diff
@@ -0,0 +1,102 @@
+*EXP.DETAILS: TERRAR01SG                                                             
+
+*FILES
+@N  EXP.DATA    AFILE        ADIR
+  1 EXP         TERRAR01.SGA                                                                                 
+@N  SPECIES     SPFILE       SPDIR
+  1 SPE         SGCER048.SPE ///Users/kristinariemer/tmp/Genotype/                                           
```

Comment:
I kind of went back and forth on this. I think we should have them there in some capacity, because it's useful to have all the example DSSAT files, both inputs and outputs. Maybe a description in the README would be a good compromise? A section about the DSSAT files, describing that the inputs ones are in the .zip, and the inputs and outputs are in this `dssat_input` subfolder, and that they're just examples and should be deleted and regenerated if someone is trying to run this themselves?

Reply:
> Maybe a description in the README would be a good compromise?

yes, that sounds good

Reply:
Added in aa7ed9f13